app/services/hyrax/custom_queries/navigators/parent_work_navigator.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Hyrax
+  module CustomQueries
+    module Navigators
+      ##
+      # Navigate from a resource to it's parent work.
+      #
+      # @see https://github.com/samvera/valkyrie/wiki/Queries#custom-queries
+      # @since 3.0.0
```

Comment:
Should this be 3.4.0?
```suggestion
      # @since 3.4.0
```